src/lib/environment/mod.rs

```diff
@@ -445,7 +449,12 @@ fn setup_env_for_project(config: &Config, crate_info: &CrateInfo) {
 }
 
 /// Sets up the env before the tasks execution.
-pub(crate) fn setup_env(cli_args: &CliArgs, config: &Config, task: &str) -> EnvInfo {
+pub(crate) fn setup_env(
+    cli_args: &CliArgs,
+    config: &Config,
+    task: &str,
+    home: Option<PathBuf>,
```

Comment:
Seeing that `home` won't be used anywhere, I just consumed it.
I can make this a reference if needed.